examples/multiple-choice/index.html

```diff
@@ -41,8 +41,8 @@
       [
         {
           question: 'What framework should we start with?',
-          answers: ['React', 'Ember.js', 'Vue'],
-          answer: 1
+          answers: ['React', 'Ember.js', 'Vue', 'Angular', 'jQuery'],
```

Comment:
incidentally, also many of these aren't mutually exclusive, so maybe there should be an attribute for switching this component into a multi-option mode